stages/org.osbuild.systemd.unit.create

```diff
@@ -44,6 +46,18 @@ SCHEMA = r"""
         "type": "string",
         "pattern": "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.service$"
     },
+    "unit-type": {
+        "type": "string",
+        "enum": ["system", "global"],
+        "default": "system",
+        "description": "Selects between systemd system or global unit to add dropin"
```

Comment:
Thanks for keeping consistency here (even though I would also prefer this to be named "user" it's too late). However I think it would be nice to add a hint in the description, something like: `Selects between systemd system or global (user) unit to add dropin` or similar. Maybe even in both places (but we (osbuild team) can also do that in a followup).